dashboard/src/api/hooks/useGetProject.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import { Project } from '../../interfaces/projects';
+import client from '../client';
+import '../mocks/useGetProject.mock';
+
+const getProject = async (projectId?: number) => {
+  const response = await client.get<Project>(
+    `/api/v1/projects/${projectId}`,
```

Comment:
make "/api/v1/projects" as an exportable constants, because it is used different places.

Reply:
we tried to add `/api/v1/` to the baseURL in axios setup but then we came up the a conclusion that our current way of handling it is easier to understand and keep the same both in mocked and non-mocked environments